docs/integrations/mlflow.rst

```diff
@@ -3,76 +3,163 @@
+	
+.. _mlflow-quickstart:
 
-.. code-block:: bash
+Quickstart
+----------
+
+This quickstart will show you how to:
+
+* Install the necessary Neptune packages
+* Convert MLruns directory of experiment runs into Neptune experiments.
+
+|Run on Colab|
+
+.. _mlflow-before-you-start-basic:
 
-    pip install neptune-mlflow
+Before you start
+^^^^^^^^^^^^^^^^
+#. This integration requires you to have your Personal API token. You need a Neptune account to have this. Create one for free |neptune-register| if you haven't already
 
-Sync your MLruns with Neptune
------------------------------
+#. Ensure that you have ``Python 3.x`` and following libraries installed:
 
-1. Set your ``NEPTUNE_API_TOKEN``:
+   * ``neptune-mlflow``
+   * ``mlflow==1.12.1``. See |mlflow-install|.
+   
+   .. code-block:: bash
+   	
+      pip install --quiet mlflow neptune-mlflow
 
-Linux/IOS:
+#. You also need minimal familiarity with Mlflow. Have a look at the |mlflow-guide| guide to get started.
+
+Step 1: Set your ``NEPTUNE_API_TOKEN``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Linux/iOS:
 
 .. code:: bash
 
-    export NEPTUNE_API_TOKEN='YOUR_NEPTUNE_API_TOKEN'
+    export NEPTUNE_API_TOKEN='YOUR_API_TOKEN'
 
 Windows:
 
 .. code-block:: bat
 
-    set NEPTUNE_API_TOKEN="YOUR_NEPTUNE_API_TOKEN"
+    set NEPTUNE_API_TOKEN="YOUR_API_TOKEN"
+
+.. tip::
+
+    Read more about how to :ref:`securely set the Neptune API token <how-to-setup-api-token>`.
 
-2. Navigate to the MLflow project in your directory and run:
+Step 2: Sync your MLruns with Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Navigate to the MLflow project in your directory and run:
 
 .. code-block:: bash
 
     neptune mlflow --project USER_NAME/PROJECT_NAME
 
-Alternatively you can point to the MLflow project directory:
+Alternatively, you can point to the MLflow project directory:
 
 .. code-block:: bash
 
-    neptune mlflow /PATH/TO/MLflow_PROJECT --project WORKSPACE_NAME/PROJECT_NAME
+    neptune mlflow /PATH/TO/MLflow_PROJECT --project USER_NAME/PROJECT_NAME
 
-**That's it! You can now browse and collaborate on your MLflow runs in Neptune.**
+.. tip::
+
+    You can also set the project qualified name in the ``NEPTUNE_PROJECT`` environment variable and skip the ``--project`` argument
+	
+	Linux/macOS:
+
+	.. code:: bash
 
-Organize and share your MLflow experiments
-------------------------------------------
+		export NEPTUNE_PROJECT='USER_NAME/PROJECT_NAME'
 
-.. image:: ../_static/images/integrations/mlflow_1.png
-   :target: ../_static/images/integrations/mlflow_1.png
-   :alt: organize MLflow experiments in Neptune
+	Windows:
 
+	.. code-block:: bat
 
-.. image:: ../_static/images/integrations/mlflow_2.png
-   :target: ../_static/images/integrations/mlflow_2.png
-   :alt: share artifacts logged during MLflow run
+		set NEPTUNE_PROJECT="USER_NAME/PROJECT_NAME"
+
+**That's it! You can now browse and collaborate on your MLflow runs in Neptune.**
+
+.. image:: ../_static/images/integrations/mlflow.png
```

Comment:
Please add a screenshot with a few of the experiments (not just one). 
You could perhaps do it in a compare mode even -> not sure your call on what looks more interesting.

Reply:
Added both :)